pkg/controller/v1alpha1/localmodelnode/file_utils_test.go

```
@@ -112,6 +112,49 @@ func TestFileSystemHelper_removeModel(t *testing.T) {
+	}
+	defer os.Chmod(brokenDir, 0o755) //nolint
+
+	if !helper.hasPermissionIssues() {
+		t.Errorf("expected permission issues when a subdir is inaccessible")
+	}
+
+	// Case 4: Non-directory entries are ignored
+	os.Chmod(brokenDir, 0o755) //nolint
+	os.RemoveAll(brokenDir)    //nolint
+	filePath := filepath.Join(tempDir, "regular-file")
+	if err := os.WriteFile(filePath, []byte("data"), 0o000); err != nil { //nolint
```

golangci-lint (2.12.2)

[error] directive `//nolint` is unused

(nolintlint)